### ISSUE TITLE
Improve the round metadata 

### ISSUE THREAD

**Issue by thelostone-mc** [gitcoinco/grants-stack#1656]
I saw that we're working on a Round settings page, if we're going to work on this area I wanted to start a discussion on the metadata format and raise some issues I noticed, it could be a good opportunity to improve it.

At the moment this is how the quadraticFundingConfig value looks like:

{
   "matchingFundsAvailable":1000,
   "matchingCap":true,
   "minDonationThreshold":true,
   "sybilDefense":true,
   "matchingCapAmount":10,
   "minDonationThresholdAmount":1
}

I think there are a couple of issues with it

- matchingCapAmount is actually a percentage and not an amount, we should either rename it to matchingCapPercentage or store an actual amount in wei
- minDonationThresholdAmount and minDonationThreshold are redundant because minDonationThresholdAmount can just be 0 to disable it
- minDonationThresholdAmount should be a wei big int, not a number
- matchingFundsAvailable is a duplicate of the RoundImplementation.matchAmount property, do they serve different purposes? which one is the source of truth? at the moment in the indexer we're using the matchAmount in the contract since it has events to update it, we should have only one to avoid confusion

any thoughts? is this something we can do now with the round settings page or is it a separate tech debt task?

**Comment by hmrtn** [comment]
We should have a x-team discussion about what _exactly_ needs to be altered here before proceeding, as this change is not trivial.

**Comment by boudra** [comment]
@hmrtn can you come up with a suggestion looking at the description of the issue to move this forward?

**Comment by hmrtn** [comment]
From what you said I think this data schema is appropriate.
```
{
  "hasSybilDefense": Bool,
  "matchingCapPercentage": Float
  "minDonationThresholdAmount": BigInt
}

```

**Comment by boudra** [comment]
I agree we should store the matchingCap as an amount and not percentage.

for the passport option I would go for `enableSybilDefense` or `sybilDefenseEnabled`, what do you think?

**Comment by hmrtn** [comment]
@gravityblast @thelostone-mc What do you guys think of the proposed schema? Would like your input if anything is missing. 

I am being deliberately conscientious here because this will likely require altering quite a number of files and their dependent usages.

**Comment by gravityblast** [comment]
LGTM!

**Comment by boudra** [comment]
@hmrtn just fixed a bug in the calculator where were treating the min amount as a token value and didn't realize it was USD, can we name it something like:

```
minAmountUSD
```

so it's clear it's USD to the next person who looks at this

**Comment by boudra** [comment]
@hmrtn it's best to go ahead with implementation and iterate during PR review if someone has something to add, worst case scenario you have to rename a few things but I think in this case it should be OK, waiting for feedback here just holds up the task, from the top of my head I think the only places we need to change here is the settings page which I believe you have the context there since you worked on it and the calculator.

**Comment by thelostone-mc** [comment]
@boudra I'd lean towards stoing matchingCapPercentage (we could store the amount as well)
Reason: On the round edit flow ->  we'd want to show the exact same % the round operator had set when creating the round

**Comment by boudra** [comment]
that makes sense @thelostone-mc !

**Comment by boudra** [comment]
This is blocked because we need to update the indexer, and the indexer is frozen because we haven't finalized the beta rounds yet. Once we finalize we can deploy the new indexer and deploy this change too.

**Comment by boudra** [comment]
Blocked by: https://github.com/gitcoinco/allo-indexer/issues/122

**Comment by michellema1208** [comment]
@gravityblast is this something we should still prioritize/finish?
